src/job-exporter/src/docker_inspect.py

```diff
@@ -25,7 +25,7 @@
 
 logger = logging.getLogger(__name__)
 
-target_label = {"PAI_HOSTNAME", "PAI_JOB_NAME", "PAI_USER_NAME", "PAI_CURRENT_TASK_ROLE_NAME", "GPU_ID"}
+target_label = {"PAI_JOB_NAME", "PAI_USER_NAME", "PAI_CURRENT_TASK_ROLE_NAME", "GPU_ID"}
```

Comment:
Can we use these as env to find k8s docker container?

Reply:
what do you mean?